NOTES.md

```diff
@@ -1,3 +1,186 @@
+# Release 2.0.0-RC1 (Musashi)
+
+Released on 2015-03-23
+
+This is a release candidate for the upcoming RethinkDB 2.0 release.
+It is not for production use. It might still have bugs. If you decide
+to test it, please [back up all][backup-docs] your data first.
+
+Release highlights:
+* Support for attaching a changefeed to the `get_all` and `union` commands
+* Improved support for asynchronous queries
+* Added a new feature called `tables` which can be used to store JSON documents
+
+Read the [blog post][2.0-RC1-blog] for more details.
+
+[2.0-RC1-blog]: http://rethinkdb.com/blog/2.0-release-candidate/
+
+## Compatibility ##
+
+This release may not be compatible with RethinkDB 2.0.
```

Comment:
Could you elaborate on this? "may not be compatible" is a pretty broad statement.